CHANGELOG.md

```diff
@@ -69,6 +69,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * The new template keywords `local_branches`/`remote_branches` are added to show
   only local/remote branches.
 
+* `jj workspace add` now preserves all parents of the old working-copy commit
```

Comment:
Is this a new feature or a fixed bug?

Reply:
I consider the current behavior an oversight. Or perhaps it was poor judgment :)

Reply:
I'll mark this resolved, but let me know if you disagree. We can change it in this PR or later if necessary.